composer.json

```diff
@@ -9,13 +9,12 @@
         }
     ],
     "require": {
-        "php": "^5.3.3|^7.0",
+        "php": ">=7.2.5",
```

Comment:
How about not using unbounded version constraints here?

Also see https://github.com/symfony/symfony/pull/36876#issuecomment-631351947.

Reply:
https://twitter.com/nicolasgrekas/status/1263023258938548225

https://twitter.com/driesvints/status/1263031553623556099

Reply:
Got bitten by this yesterday on one of my dep upgrades, and got something ancient because it had `>=7.2` in its constraints.

Let's please not repeat the same mistakes we did with un-bounded dependency ranges in 2012~2013 :-)

Reply:
So, you mean we should add ^8.0.0 too?

Reply:
Correcto 👍

Reply:
Done @ocramius